src/functTest/groovy/com/bmuschko/gradle/docker/AbstractFunctionalTest.groovy

```diff
@@ -56,6 +56,7 @@ abstract class AbstractFunctionalTest extends Specification {
             .withArguments(arguments + '-s' as List<String>)
             .withPluginClasspath()
             .withEnvironment(envVars)
+            .forwardOutput()
```

Comment:
This was probably only added temporarily for troubleshooting purposes. It usually produces a lot of output. I will remove it later.

Reply:
Actually it was intended.
I was majorly confused by not getting any helpful output.
But as you like. :-)